drivers/mei/pkg.yaml

```diff
@@ -0,0 +1,34 @@
+name: mei
+variant: scratch
+shell: /toolchain/bin/bash
+dependencies:
+  - stage: base
+  # The pkgs version for a particular release of Talos as defined in
+  # https://github.com/siderolabs/talos/blob/<talos version>/pkg/machinery/gendata/data/pkgs
+  - image: "{{ .BUILD_ARG_PKGS_PREFIX }}/kernel:{{ .BUILD_ARG_PKGS }}"
+steps:
+  - prepare:
+      - |
+        sed -i 's#$VERSION#{{ .VERSION }}#' /pkg/manifest.yaml
+  - install:
+      - |
+        export KERNELRELEASE=$(find /lib/modules -type d -name "*-talos" -exec basename {} \+)
+
+        mkdir -p /rootfs
+
+        xargs -a /pkg/files/modules.txt -I {} install -D /lib/modules/${KERNELRELEASE}/{} /rootfs/lib/modules/${KERNELRELEASE}/{}
```

Comment:
I guess we need to make this run for amd64 only, as this is not applicable (and we have no modules) for arm64 to make the extension "empty" for arm64

Reply:
but the step doesn't seem to fail, so I guess it's fine :thinking:

Reply:
is the CI pulling `pkgs` from `main`? If not I would actually expect both to fail since there hasn't been a `pkgs` release with the necessary modules.

Reply:
oh it's not, https://github.com/siderolabs/extensions/blob/main/.kres.yaml#L56 needs update here, weird it passed at all

Reply:
oh I know, `make rekres` was not run, so the targets was not added to `Makefile`

Reply:
OK, I ran that and took a swing at gating on arch. I don't have a complete understanding of the build architecture, so I made an educated guess that Docker's `TARGETARCH` would be available to the install script.

Reply:
i'll fix this up